themes/default/content/docs/intro/console/organizations.md

```diff
@@ -50,6 +50,20 @@ For example, to become a member of a Pulumi organization backed by a GitLab Grou
 you must associate a GitLab identity with your Pulumi account, and also
 be a member of that GitLab group.
 
+## Inviting Members to an Organization
+
+Pulumi organization administrators can invite new members to an organization.
+
+To invite a new member:
+
+1. Select the members tab.
+2. To invite new members using their email addresses, select **Invite members**. After the new member(s) have been invited, they will receive an email with a link to click to accept the invitation.
+3. To invite new members by sharing a link, select **Copy new invite link**. This will copy a link to your clipboard that you can share directly. 
```

Comment:
```suggestion
3. To invite a new member by sharing a link, select **Copy new invite link**. This will copy a link to your clipboard that you can share directly. 
```